styles/styles.css

```diff
@@ -0,0 +1,389 @@
+* {box-sizing: border-box;}
+
+* {font-family: Georgia, serif;}
+
+
+header {
+  display: flex;
+
+  flex-direction: column;
+  padding: 2em;
+
+  background: rgb(115,136,153);
+  background: linear-gradient(0deg, rgba(115,136,153,1) 0%, rgba(9,60,102,1) 100%);
+  background-size: auto;
+  background-image: url('../images/header-image.jpg');
+
+
+
+}
+
+
+.logo {
+  width: 70%;
+}
+
+.logo img {
+  width: 60%;
+}
+
+.logo h1 {
+  font-size: 2em;
+}
+
+header button {
+  display: flex;
+  width: 10em;
+  justify-content: center;
+  padding: .5em;
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+}
+
+.checkout {
+  justify-content: center;
+  flex-direction: column;
+  align-self: flex-end;
+  display: flex;
+}
+
+i {
+  color: white;
+  font-size: 1em;
+  display: flex;
+  align-self: flex-end;
+  margin-bottom: 1em;
+}
+
+body {
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+nav {
+  background-color: #e1e7eb;
+  width: 100%;
+  padding: 1.5em;
+}
+
+nav li {
+display: inline;
+}
+
+nav a {
+ text-decoration: none;
+ font-weight: bold;
+ color: #18387d;
+ font-size: 1.2em;
+
+}
+h1 {
+  font-style: italic;
+  font-size: 1.5em;
+  color: #73c74a;
+}
+
+
+section p {
+  font-size: 1.5em;
+}
+
+
+.lead-article {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-end;
+
+  margin-top: 0;
+
+}
+
+h2 a {
+  text-decoration: none;
+  color: #18387d;
+
+}
+
+h5 a {
+  text-decoration: none;
+  color: #29b1cc;
+  font-size: 1.2em;
+}
+
+aside h5 {
+  padding-bottom: 15%;
```

Comment:
Generally I like REM's or EM's for padding, as it's usually smaller values. And in this case I'd use margin instead of padding as you're putting space between elements. And a REM or EM would be a bit more of a stable way of spacing elements.